addons/crm/wizard/crm_lead_to_opportunity_mass_views.xml

```diff
@@ -32,12 +32,18 @@
                         </list>
                     </field>
                 </group>
-                <group invisible="name != 'convert'" string="Customers" col="1">
-                    <field name="action" class="oe_inline" widget="radio"/>
-                    <group col="2">
+                <group string="Customers">
+                    <group invisible="name != 'convert'">
+                        <field name="action" class="oe_inline" widget="radio"/>
+                    </group>
+                    <group>
+                        <field name="company_partner_id"
```

Comment:
So I went to a lead with some value for Company Name, Shufflebeat, here (being, according to helper `The name of the future partner company that will be created while converting the lead into opportunity` ) and when opening the wizard we do not show the name of the company anywhere. But we suggest to link to a company, with the field being empty. I feel there is something rather confusing functionally (should we prefill the field or reword?).

![image](https://github.com/user-attachments/assets/516d161c-bd4a-42c0-9821-29cf177d284d)